mage_ai/data_preparation/repo_manager.py

```diff
@@ -248,3 +224,46 @@ def update_project_uuid():
 
 def get_project_uuid() -> str:
     return project_uuid
+
+
+# These should not be used. Please use the corresponding functions in
+# mage_ai/settings/__init__.py
+
+REPO_PATH_ENV_VAR = 'MAGE_REPO_PATH'
+
+
+def get_repo_path() -> str:
+    warn(
+        'repo_manager.get_repo_path is deprecated. Please use mage_ai.settings.repo.get_repo_path',
+        DeprecationWarning,
+        stacklevel=2
+    )
+    return os.getenv(REPO_PATH_ENV_VAR) or os.getcwd()
```

Comment:
You can actually call the method from `mage_ai.settings.repo` here for code consistency.

Reply:
good idea